opre_ops/ops_site/helpers/csv_to_json.py

```diff
@@ -0,0 +1,45 @@
+import csv
```

Comment:
FYI. This might need to be customized depending on the model/fixture data output, but I tried to make it as generic as I could without being overly cumbersome and complicated

. It's also not meant to write out to the fixture file, just to output formatted JSON to a specified output file that the developer can then paste into the fixture file.

Reply:
[suggestion] We should write a test or two for this.  May not need to happen in this PR

Reply:
Hmm, I'd be curious to hear more @amymok about the value of testing this. My gut feel is that a script we use locally to load in fake/sample data doesn't necessarily need tests, but open to being convinced otherwise! 

If we do write tests for this script I'd suggest they happen in a future PR like you mentioned Amy.

Reply:
I also went back on forth on whether or not to write tests for this and also concluded that, given we won't use it anywhere except local, it didn't make sense to write tests. But I can also be convinced otherwise!

Reply:
Sorry I deleted my comment bc I thought it was a wrong place to add it, but adding it back in now:

>[suggestion] We should write a test or two for this. May not need to happen in this PR

Reply:
Okay I see the point here.  So the use of the script is really to generate data, and we will eyeball the data before using it anyways, so it seems unnecessary for that purpose?

Reply:
I think that makes sense.  I am okay with no test here, but agree with @alexsoble on some more comments or some README type of documentation on the purpose of this file, so other developer will know this is for them to use to create fixture